tests/setup.py

```diff
@@ -13,14 +13,13 @@
     package_data={'': ['*.md']},
     include_package_data=True,
     install_requires=[
-        'ducktape@git+https://github.com/redpanda-data/ducktape.git@cad0aeaa32da8ad43a297e50d6cdb215770fd10c',
-        'prometheus-client==0.9.0', 'pyyaml==6.0', 'kafka-python==2.0.2',
-        'crc32c==2.2', 'confluent-kafka==2.2.0', 'zstandard==0.15.2',
-        'xxhash==2.0.2', 'protobuf==4.21.8', 'fastavro==1.4.9',
-        'falcon==3.1.3', 'psutil==5.9.0', 'numpy==1.22.3', 'pygal==3.0',
-        'pytest==7.1.2', 'jump-consistent-hash==3.2.0',
-        'azure-storage-blob==12.14.1',
-        'kafkatest@git+https://github.com/apache/kafka.git@3.2.0#egg=kafkatest&subdirectory=tests',
+        'ducktape@git+https://github.com/redpanda-data/ducktape.git@7772220005a27957d3a796de0e19d3c51eb61f6a',
```

Comment:
Nit: using a tag would be really nice (we can add our own tag), for readability and for indicating immutability.

Reply:
we'll move to using tags after we simplify the fork